http/headers/Content-Security-Policy.json

```
@@ -1018,6 +1018,40 @@
               }
             }
           },
+          "inline-speculation-rules": {
+            "__compat": {
+              "description": "Source expression allowing inline speculation rules",
```

This is a bug for wasm-unsafe-eval as well, I think. We should have the keyword in the description so that it is better recognizable. 

```suggestion
              "description": "<code>inline-speculation-rules</code> source expression",
```